docs/arcadia-users-group/20230228-intro-to-python-1/lesson.md

```diff
@@ -0,0 +1,548 @@
+# Introduction to Python, Part 1
+
+Follow along in **Binder** by clicking the badge below:  
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Arcadia-Science/arcadia-computational-training/das/pytutorial?labpath=docs%2Farcadia-users-group%2F20230228-intro-to-python-1%2Flesson.ipynb)
```

Comment:
should the binder open to the launch screen so learners can open a blank notebook? I often find that coding along is really useful -- so instead of having everything pre-filled, each learner would type each cell as you type them

Reply:
That make sense! I'll look into how to do that.

Reply:
I added a `playground.ipynb` which links to the `lesson.ipynb` as the default notebook for Binder!  

I also updated the Binder link so it should point to where the `playground.ipynb` will live in `main` once this is merged. Hopefully it works but I can't test it quite yet!

Reply:
ok we can check right after this is merged and patch as necessary